wire-ios/Wire-iOS/Sources/UserInterface/Location/LocationSelectionViewController.swift

```diff
@@ -73,81 +98,95 @@ final class LocationSelectionViewController: UIViewController {
-            annotationView.centerXAnchor.constraint(equalTo: mapView.centerXAnchor, constant: 8.5),
-            annotationView.bottomAnchor.constraint(equalTo: mapView.centerYAnchor, constant: 5),
-            annotationView.heightAnchor.constraint(equalToConstant: 39),
-            annotationView.widthAnchor.constraint(equalToConstant: 32),
-
-            locationButton.leadingAnchor.constraint(equalTo: view.leadingAnchor, constant: 16),
-            locationButton.bottomAnchor.constraint(equalTo: sendController.topAnchor, constant: -16),
-            locationButton.widthAnchor.constraint(equalToConstant: 28),
-            locationButton.heightAnchor.constraint(equalToConstant: 28)
+            annotationView.centerXAnchor.constraint(equalTo: mapViewController.mapView.centerXAnchor, constant: LayoutConstants.annotationViewCenterXOffset),
+            annotationView.bottomAnchor.constraint(equalTo: mapViewController.mapView.centerYAnchor, constant: LayoutConstants.annotationViewBottomOffset),
+            annotationView.heightAnchor.constraint(equalToConstant: LayoutConstants.annotationViewHeight),
+            annotationView.widthAnchor.constraint(equalToConstant: LayoutConstants.annotationViewWidth),
+
+            locationButton.leadingAnchor.constraint(equalTo: view.leadingAnchor, constant: LayoutConstants.locationButtonLeadingOffset),
+            locationButton.bottomAnchor.constraint(equalTo: sendController.topAnchor, constant: LayoutConstants.locationButtonBottomOffset),
+            locationButton.widthAnchor.constraint(equalToConstant: LayoutConstants.locationButtonWidth),
+            locationButton.heightAnchor.constraint(equalToConstant: LayoutConstants.locationButtonHeight)
         ])
     }
 
-    @objc fileprivate func locationButtonTapped(_ sender: IconButton) {
+    // MARK: - User Actions
+
+    private func locationButtonTapped() {
         zoomToUserLocation(true)
     }
 
-    fileprivate func updateUserLocation() {
-        mapView.showsUserLocation = userLocationAuthorized
+    // MARK: - Helpers
+
+    private func updateUserLocation() {
+        mapViewController.mapView.showsUserLocation = userLocationAuthorized
         if userLocationAuthorized {
-            locationManager.startUpdatingLocation()
+            appLocationManager.startUpdatingLocation()
         }
     }
 
-    fileprivate func zoomToUserLocation(_ animated: Bool) {
+    private func zoomToUserLocation(_ animated: Bool) {
         guard userLocationAuthorized else { return presentUnauthorizedAlert() }
-        let region = MKCoordinateRegion(center: mapView.userLocation.coordinate, latitudinalMeters: 50, longitudinalMeters: 50)
-        mapView.setRegion(region, animated: animated)
+        let region = MKCoordinateRegion(center: mapViewController.mapView.userLocation.coordinate, latitudinalMeters: 50, longitudinalMeters: 50)
+        mapViewController.mapView.setRegion(region, animated: animated)
```

Comment:
The PR's description says:
> Created MapViewController class that encapsulates the MKMapView and its configurations.

So I am asking myself if more logic should be moved to the `MapViewController` to avoid explicit usage of types like `MKCoordinateRegion` outside.

Reply:
Done: https://github.com/wireapp/wire-ios/commit/4d243f4f3027aa40b2855efc4b5ac38e95552965